### DISCUSSION TITLE
Preserve form after user submission

### DISCUSSION THREAD

**Discussion by Naatan** [charmbracelet/huh#585] [Ideas]
**Is your feature request related to a problem? Please describe.**
I want the form to remain visible after the user has provided their input. At the moment it drops the form, which makes for a confusing user experience.

**Describe the solution you'd like**
Some sort of option I can toggle to have it not delete the rendered form when it is done.

**Describe alternatives you've considered**
Open to other solutions so long as they achieve the desired outcome.

**Additional context**
Dug through code and docs but could not find any obvious way to achieve what I am describing.

**Comment by MatthewWaring** [comment]
@Naatan 
Have you tried using the`.WithAccessible(true)`  feature on huh forms? This will preserve the form and drop to new lines on the terminal

Using that will result in the following:
```go
package main

import (
	"log"

	"github.com/charmbracelet/huh"
)

func main() {
	form := huh.NewForm(
		huh.NewGroup(
			huh.NewSelect[string]().
				Options(huh.NewOptions("Italian", "Greek", "Indian", "Japanese", "American")...).
				Title("Favorite Cuisine?"),
		),

		huh.NewGroup(
			huh.NewInput().
				Title("Favorite Meal?").
				Placeholder("Breakfast"),
		),
	).WithAccessible(true)

	err := form.Run()
	if err != nil {
		log.Fatal(err)
	}
}
```

The following above will result in the previous forms being saved along with the user input and it dropping to new lines
![image](https://github.com/charmbracelet/huh/assets/139682815/4ff64a3e-9883-4389-a23c-8b157777e25b)

**Comment by bashbunni** [comment]
Hey @Naatan, thank you so much for wanting to make this project better! I've converted this to a discussion so we can talk through it and keep our backlog clear. If it does end up being something we want to add, we'll promote it back to an issue when we are ready to work on it.

If we commit to this feature, that's a feature we will need to maintain for the lifespan of this project. So, since "Yes is forever", we are carefully considering new features.

**Comment by gurleensethi-docker** [comment]
It would be nice to have this feature, be able to preserve the output of the form as is, helps the user keeps track of what the input was.
